### PR TITLE
Make the app look a little bit better

### PR DESCRIPTION
**Pull request by nickkossolapov**
Sorry for the big PR

image

image

image

### CODE REVIEW

File: src/components/shared/downloadInput/UploadInput.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { useAppDispatch } from "../../../app/hooks"
+import { setPlaybackFile } from "../../../app/playbackFileSlice"
+import { clearSentence } from "../../../app/sentenceSlice"
+import styles from "./DownloadInput.module.css"
+
+export interface DownloadInputProps {
+  id: string
+}
+
+export function UploadInput(props: DownloadInputProps) {
```
Comment (Regan-Koopmans) [inline src/components/shared/downloadInput/UploadInput.tsx:11]: This is more of a general comment, but I see a mixture of terminology. Is this a download input or an upload input? I can see how both could work, but I think we just need to decide on one.

File: src/components/shadowing/shadowingLayout/ShadowingLayout.module.css [hunk 2]
```diff
@@ -33,3 +36,10 @@
 .dialogue p {
   margin: 0;
 }
+
+.newAudio {
+  composes: squareButton from "../../styles/buttons.module.css";
```
Comment (Regan-Koopmans) [inline src/components/shadowing/shadowingLayout/ShadowingLayout.module.css:41]: Haven't seen this syntax before, it's really cool

**Review by Regan-Koopmans — COMMENTED** [review]
Really awesome work. I just have a general comment on naming, but otherwise I think the code is good 👍

File: src/components/shared/downloadInput/UploadInput.tsx (hunk 1 above)
Comment (nickkossolapov) [reply src/components/shared/downloadInput/UploadInput.tsx:11]: Good catch. I started to rename this to match the UI, but forgot to go back and update the old code

File: src/components/shadowing/shadowingLayout/ShadowingLayout.module.css (hunk 2 above)
Comment (nickkossolapov) [reply src/components/shadowing/shadowingLayout/ShadowingLayout.module.css:41]: This is only part of CSS modules though, not proper CSS

**Pull request merged**